chat/chat-tool/tool/src/webapp/jsp/roomMonitor.jspf

```diff
@@ -42,7 +42,7 @@
             <c:out value="<span class='chatUserOnline'></span>" escapeXml="false" />
             <c:out value="</span>" escapeXml="false" />
 
-            <c:out value="<span class='chatNameDate'><span class='chatName'>${message.owner}</span>" escapeXml="false" />
+            <c:out value="<span class='chatNameDate'><span class='chatName'>${fn:escapeXml(message.owner)}</span>" escapeXml="false" />
```

Comment:
I am not familiar with this templating language, but it seems to me this should be written as follows:

    <span class='chatNameDate'><span class='chatName'><c:out value="${message.owner}"></span>